charts/ceph-csi-cephfs/values.yaml

```diff
@@ -191,6 +191,56 @@ topology:
     - failure-domain/region
     - failure-domain/zone
 
+storageClass:
+  # Specifies whether the Storage class should be created
```

Comment:
provisioner name is missing in  the storageclass

Reply:
The Provisioner name is picked up from the drivername in the values.yaml